signac/contrib/utility.py

```diff
@@ -256,6 +385,18 @@ def __hash__(self):
 
 
 def _to_hashable(l):
+    """Create a hash of passed type.
+
+    Parameters
+    ----------
+    l :
+        Create hash for.
```

Comment:
Create a hash for what? And what are the acceptable datatypes?

Reply:
its data type will be same as x in `_encode_tree`. If you see `_to_hashable` is only called by `_encode_tree`.

Reply:
Actually this also gets called in contrib/collection.py, but you are right about the type.